src/Adapter/Module/Tab/ModuleTabRegister.php

```diff
@@ -288,11 +289,13 @@ protected function getModuleControllersFromRouting(string $moduleName): array
         }
 
         $routingControllers = [];
-        $moduleRoutes = $this->routingConfigLoader->import($routingFile, 'yaml');
-        foreach ($moduleRoutes->getIterator() as $route) {
-            $legacyController = $route->getDefault('_legacy_controller');
-            if (!empty($legacyController)) {
-                $routingControllers[] = $legacyController;
+        if ($this->routingConfigLoader instanceof Loader) {
```

Comment:
routingConfigLoader is already a LoaderInterface and is a mandatory parameter :thinking:

Reply:
But the method `import` is only declared on abstract class `Loader`.

Reply:
So we need a Loader, and not a LoaderInterface :/

Reply:
It adds a BC but it's logical.

Reply:
I documented it ;)